src/setup-node.ts

```diff
@@ -17,6 +19,15 @@ async function run() {
       await installer.getNode(version);
     }
 
+    // Output version of node and npm that are being used
+    const nodePath = await io.which('node');
+    const nodeVersion = cp.execSync(`${nodePath} --version`);
+    console.log(`Node Version: ${nodeVersion}`);
+
+    const npmPath = await io.which('npm');
+    const npmVersion = cp.execSync(`${npmPath} --version`);
+    console.log(`npm Version: ${npmVersion}`);
```

Comment:
not sure why none of this would fail.  On one hand I was thinking we should make all this best effort.  But then I rethought that if this can't work, then the action didn't do it's job so it should fail.  So I think it's fine how it is.